services/database/class.ts

```diff
@@ -0,0 +1,66 @@
+import prisma from "@database";
+import { Class } from "@prisma/client";
+import { createClassInput } from "models/Class";
```

Comment:
Is it generally better convention to have these input types be capitalized or lowercase at the start? I went with uppercase since that seems to be the case for all the types defined in `@prisma/client` and we should be consistent

Reply:
I'm not really sure what the convention is, perhaps @mahadzaryab1 could give more thoughts

Reply:
yeah I think types should be Pascal Case

Reply:
here's a good style guide - https://basarat.gitbook.io/typescript/styleguide

Reply:
@jason17h